index.html

```diff
@@ -158,8 +158,15 @@
            observer.disconnect();
       }
     });
-    // subscribe to Resource-Timing and User-Timing events
-    observer.observe({entryTypes: ['resource', 'mark', 'measure']});
+    observer.observe({
+      // subscribe to Resource-Timing, User-Timing, and Server-Timing events
+      entryTypes:
+        ['resource', 'mark', 'measure', 'server'],
+
+      // retrieve the buffered Resource-Timing, User-Timing, and Server-Timing events
+      bufferedTypes:
+        ['resource', 'mark', 'measure', 'server']
```

Comment:
If we go down this route then we're encouraging developers to create observers that are never disconnected, which is counter to all other observer interfaces. Also, note that the use case you're describing is already covered by querying `performance.getEntriesByType` within onload event; I don't think we need to invent a new mechanism for this.

The other benefit of a boolean is that you don't have to repeat yourself twice in the constructor. My guess is that in majority of cases the set of entries you care about going forward, and from the past, is the same.